can/io/canutils.py

```diff
@@ -164,8 +170,16 @@ def on_message_received(self, msg):
         else:
             framestr += " %03X#" % (msg.arbitration_id)
 
+        if msg.is_error_frame:
+            eol = "\n"
+        else:
+            if msg.is_rx:
+                eol = " R\n"
+            else:
+                eol = " T\n"
```

Comment:
It's shorter and still well-readable:

```suggestion
            eol = " R\n" if msg.is_rx else " T\n"
```

Reply:
Done